ecosystem/sep-0038.md

```diff
@@ -54,21 +101,21 @@ No request arguments required.
 
 Name | Type | Description
 -----|------|------------
-`assets` | array | An array of objects describing the Stellar assets available in exchange for one or more of the `currencies` listed.
-`currencies` | array | An array of objects describing the off-chain currencies available in exchange for one or more of the `assets` listed.
+`stellar` | array | An array of objects describing the Stellar assets available in exchange for one or more of the assets listed under other namespaces.
+`iso4217` | array | An array of objects describing the fiat currencies available in exchange for one or more of the `stellar` assets listed.
```

Comment:
❓ Could this be a single array, given that the values will indicate if they are stellar, or iso4217, or some other type? If they need to be separated then I think we should rename iso4217 here to 'other' or 'external' so that it will contain whatever other scheme in its values and not only iso4217 currencies.

Reply:
It could be a single array, but I think its helpful to separate `stellar` from `external` assets because an asset in one group can only be exchanged for an asset in the other group, and putting all objects in a single array does not communicate this distinction.

I don't feel strongly about this though.

Reply:
On second thought, i think using a single list is acceptable, because we still have the `GET /prices` endpoint that provides the assets that can be exchanges for an asset provided in the request.